Nodejs/Product/Nodejs/NodejsPackage.cs

```diff
@@ -248,6 +252,23 @@ internal sealed partial class NodejsPackage : CommonPackage {
             _logger.LogEvent(NodejsToolsLogEvent.AnalysisLevel, IntellisenseOptionsPage.AnalysisLevel);
         }
 
+        private void InitializeTelemetry() {
+            if (_telemetryLogger == null) {
+                // Get telemetry logger
+                _telemetryLogger = TelemetrySetup.GetLogger();
+
+                // Set common properties to be sent with all ntvs events
+                var versionInfo = FileVersionInfo.GetVersionInfo(typeof(NodejsPackage).Assembly.Location);
+                // NTVS version
+                _telemetryLogger.SetCommonProperty(new DataPoint(TelemetryProperties.NtvsVersion, versionInfo.ProductVersion.ToString()));
+                // VS version
+                _telemetryLogger.SetCommonProperty(new DataPoint(TelemetryProperties.HostVersion, Application.ProductVersion.ToString()));
+
+                // Log package loaded event
+                _telemetryLogger.ReportEvent(TelemetryEvents.PackageLoaded);
+            }
+        }

```

Comment:
We will also need this code when we are loading the Profiling and InteractiveWindow packages, so we should extract it out into a common method that accepts the assembly name, version info, and vs version.